data/eos-metrics-event-recorder.service.in

```diff
@@ -2,13 +2,17 @@
 Description=EndlessOS Metrics Event Recorder Server
 Requires=dbus.service
 After=dbus.service
+StartLimitIntervalSec=3600
```

Comment:
Totally arbitrary, but this seems a little long. The systemd defaults are 5 times in 10 seconds. What I'm worried about here is locking out a restart for an hour. Maybe 10 minutes?

Also, can you use non-second time units like `1h` or `10m`?

Reply:
Switched to 10 times every 10min - the intention is to limit the amount of coredumps generated in a theoretical catastrophic case where the daemon always crashes on init.